modules/ai/assets/js/editor/hooks/use-prompt.js

```diff
@@ -28,6 +28,7 @@ const usePrompt = ( fetchData, initialState ) => {
 	const [ error, setError ] = useState( '' );
 	const [ data, setData ] = useState( initialState );
 	const { updateUsagePercentage, usagePercentage } = useRequestIds();
+	const send = useRef( ( async ( payload ) => ( payload ) ) );
```

Comment:
Why do you need the ref?

Reply:
So it will have a unique instance per component